ql/src/java/org/apache/hadoop/hive/ql/exec/repl/bootstrap/load/table/LoadTable.java

```diff
@@ -297,17 +298,15 @@ static TableLocationTuple tableLocation(ImportTableDesc tblDesc, Database parent
     LOG.debug("adding dependent CopyWork/AddPart/MoveWork for table "
             + table.getCompleteName() + " with source location: "
             + dataPath.toString() + " and target location " + tgtPath.toString());
-
-    Task<?> copyTask = ReplCopyTask.getLoadCopyTask(replicationSpec, dataPath, tmpPath, context.hiveConf,
-            false, false);
+    Path moveSrcPath = performOnlyMove ? dataPath : tmpPath;
```

Comment:
What is moveSrcPath ? What is tmpPath? dataPath is on source or target? It would be good if we could rename these variables for readability

Reply:
moveSrcPath is the path used to perform move data to final warehouse dir on target.